src/Data/Int.purs

```diff
@@ -77,6 +79,21 @@ foreign import toNumber :: Int -> Number
 fromString :: String -> Maybe Int
 fromString = fromStringAs (Radix 10)
 
+-- | A type for describing whether an integer is even or odd.
+data Parity = Even | Odd
+
+derive instance eqParity :: Eq Parity
+derive instance ordParity :: Ord Parity
```

Comment:
Makes sense, although we can make it into a field without thinking of it as Z/2Z - just by thinking of addition as `xor` and multiplication as `&&`. But yeah in the absence of any concrete use case I'm inclined to leave it how it is too.